### DISCUSSION TITLE
Translate Next.js docs to other language

### DISCUSSION THREAD

**Discussion by ChanghyeonYoon** [vercel/next.js#51566] [Ideas]
### Goals

1. Translate Next.js docs to korean
2. I want to deploy and see what I've done.

### Background

The official React documentation is translated into many languages. 
There are many React developers in Korea, and many React developers use Next.js. 
For junior developers and developers who are not familiar with English, the Next.js documentation is in English, which makes it difficult. 
Therefore, we decided to start translating the Korean documentation, just as the official React documentation supports many languages.

### Proposal

I'm currently personally working on the Korean translation, and have also contributed to the [React official Korean documentation](https://ko.react.dev).
Many Koreans have already agreed and progress to participate in the translation effort. [Task Status](https://github.com/Nextjs-kr/Nextjs.ko/issues/1)

**Comment by ChanghyeonYoon** [comment]
I realize that Docs doesn't exist other languages besides English, but since it's an open source project, I'd like to ask for your review and help to ensure that the translation work of Koreans is reflected!

**Comment by leerob** [comment]
We are not currently set up to handle translations of the content for our documentation. This is something we'd like to explore in the future, though 😄

**Reply by ChanghyeonYoon** [reply]
@leerob Then can we make the document ourselves and distribute it?

**Reply by ChanghyeonYoon** [reply]
Or is there a way to build and check the translation?

**Reply by leerob** [reply]
You could keep the translated documents in a separate GitHub repo for now, forked from the main repo 👍

**Reply by ChanghyeonYoon** [reply]
Then, is there any way to build and check the changes locally?

**Reply by leerob** [reply]
To clarify, you wouldn't view the docs on `nextjs.org/docs` because the site does not yet support i18n. You would view the docs on your GitHub repo.

**Reply by ChanghyeonYoon** [reply]
How can I see it on my github repo?
Additionally, How about creating a separate homepage with subdomains like [React official documents](https://ko.react.dev) instead of using i18n? It's my personal opinion!
